metron-platform/pom.xml

```diff
@@ -46,6 +46,7 @@
 		<module>metron-enrichment</module>
 		<module>metron-solr</module>
 		<module>metron-parsers</module>
+    <module>metron-job</module>
```

Comment:
can we adjust the indentation here?

Reply:
Heh, the pom.xml has tabs instead of spaces. Rather than reformat everything in the file I just changed that line use tabs.